cmd/zc_enumerator.go

```diff
@@ -180,6 +181,10 @@ func passedFilters(filters []objectFilter, storedObject storedObject) bool {
 	if filters != nil && len(filters) > 0 {
 		// loop through the filters, if any of them fail, then return false
 		for _, filter := range filters {
+			msg, supported := filter.doesSupportThisOS()
+			if !supported {
+				glcm.Info(msg)
```

Comment:
I'm not completely sure, but I _think_ I'd lean towards this being a glcm.Error call here, instead of Info. Why? Because any time where the user gives us some parameters, and we carry on without fully respecting them, makes me nervous. The worst-case scenario would be that a user fails to see the outputted "info" message, assumes their job worked as they expected, and then does something like deleting their source files - even though what was transferred may not have matched what they asked for. Admittedly that's not super-likely here, but I still lean towards the idea that if we can't do exactly what the user asked for, we fail hard with an error.

@zezha-msft what do you think?

Reply:
I agree with the "failing hard if can't fully deliver" approach. I wasn't sure if I should use `glcm.Error` so I used `glcm.Info` as a placeholder.

Reply:
No reply from Ze yet, but given that you and I both agree @panfeng0007, let's go with glcm.Error. I checked, and we are already using that like that in other places, so its fine for you to do so here too.